Docs/sphinx/Ceptr.rst

```diff
@@ -53,13 +67,53 @@ There are three ways to use CEPTR to generate C++ mechanism files for a given ch
 
      $ ./convert.sh
 
-3. Using a helper script in the ``Models`` directory::
+   .. note:: It is possible that using this option will require for you to have a valid Cantera installed somewhere. Again, we strongly suggest using a `conda <https://conda.io/projects/conda/en/latest/user-guide/getting-started.html>`_ environment to install all required package. A simple ceptr environment can be generated using the following yaml script:: 
+
+     $ name: ceptr
+     $ channels:  
+     $ - conda-forge 
+     $ dependencies: 
+     $ - python=3.10  
+     $ - cantera
+     $ - cython  
+     $ - numpy 
```

Comment:
This seems to repeat cantera and numpy dependencies compared with  `pyproject.toml`
I'd prefer if we keep the conda install dependencies to a minimum:

Could you please try to see if creating you conda env with `conda create --name ceptr python=3.10 poetry` before calling `convert.sh` does the trick?

Reply:
As always, I can't seem to reproduce the errors that I had now that I've tried/installed all these various environments. I know for a fact that simply invoking the convert.sh from the Mechanism directory directly (LiDryer) gave me , first a python and then a cantera error. Which I solved by including cantera in the ceptr env; and which was the entire point of my modifying the instructions actually  ... if we remove that then, yes I've slightly improved the doc but I'm not addressing the main issue I encountered which is a lack of Cantera that somehow wasn't automatically taken care of.

Reply:
Clearly the solution you provided is working, but it is not ideal since `pyproject.toml` is supposed to already take care of the cantera dependency.
I just tried what I suggested on kestrel on a fresh condo env and it seems to work, but it'd be good if we can confirm that it does take care of this issue. 
@drummerdoc you mentioned you had the same problem as @EnnaDelfen , could you maybe please try what I mentioned?

Reply:
I've tried everything on another (older) mac system and ran into issues obtaining cantera via the poetry channel. This time the issue was a bit different in that the poetry update clearly indicated that cantera was not found and thus not installed. So the `conda create --name ceptr python=3.10 poetry ` alone doesn't fix the issue. Again the only fix I could find was to use the conda-forge channel to create a conda env that includes cantera. I've explained everything to Malik in private. The source of the issue may not be completely identified but the proposed note did fix all my issues so far. But you can go ahead and ignore it if you prefer it.

Reply:
Thanks for checking all that!